core/src/test/java/org/apache/iceberg/TestSnapshotJson.java

```diff
@@ -176,4 +176,41 @@ private String createManifestListWithManifestFiles(long snapshotId, Long parentS
 
     return localInput(manifestList).location();
   }
+
+  @Test
+  public void testJsonConversionSummaryWithoutOperation() {
```

Comment:
May be paranoid but I'd like to leave a mark on the test that the behavior here is out of spec, something so that readers understand this isn't a required behavior

Reply:
Nothin paranoid about that, let me add that 👍